wazo_call_logd/database/queries/queue_stat.py

```diff
@@ -42,6 +48,38 @@ def find_oldest_time(self, queue_id):
             )
             return query.scalar()
 
+    def get_stat_queues(self, tenant_uuids=None):
+        with self.new_session() as session:
+            query = session.query(
+                StatQueue.queue_id, StatQueue.name, StatQueue.tenant_uuid
+            )
+
+            if tenant_uuids:
+                query = query.filter(StatQueue.tenant_uuid.in_(tenant_uuids))
```

Comment:
See below for the code, an empty list (which mean you have access to no tenant) will result to access to all values... ?
the rule is, if you have tenant_uuids=None, then you have access to all. But if tenant_uuids=[] then you have access to nothing